.gitignore

```diff
@@ -15,4 +15,7 @@ Makefile
 .Rhistory
 .Rapp.history
 # Session Data files
-.RData
\ No newline at end of file
+.RData
+
+# silly file that alison's code makes (and shouldn't, and won't one day). you can ignore it
```

Comment:
jordan's code makes it too

Reply:
well, i wrote the vizlab code that makes your code do this =)